report/analysis.py

```diff
@@ -182,28 +181,184 @@ def _batch_bins_and_data(batches, values, function_to_apply):
     return batch_bins, data
 
 
-def make_merging_statistics_summary(dataset_statistics):
-    """Format merging statistics information into an output string."""
+formats = collections.OrderedDict(
```

Comment:
CPython 3.6+ (and Python 3.7+) dictionaries are ordered by default. No more need to use `OrderedDict`.